src/engine/routing_algorithms/routing_base_ch.cpp

```diff
@@ -71,13 +71,15 @@ void retrievePackedPathFromSingleHeap(const SearchEngineData<Algorithm>::QueryHe
 // && source_phantom.GetForwardWeightPlusOffset() > target_phantom.GetForwardWeightPlusOffset())
 // requires
 // a force loop, if the heaps have been initialized with positive offsets.
-void search(const datafacade::ContiguousInternalMemoryDataFacade<Algorithm> &facade,
+void search(SearchEngineData<Algorithm> & /*engine_working_data*/,
+            const datafacade::ContiguousInternalMemoryDataFacade<Algorithm> &facade,
```

Comment:
yes, `SearchEngineData<Algorithm>` contains  mutable heaps, `facade` contains constant data.

It is the same convention as before: `SearchEngineData<Algorithm> &engine_working_data` just hides `heap_2` from the parameters list.